patches/net/minecraft/client/renderer/entity/layers/HumanoidArmorLayer.java.patch

```diff
@@ -13,6 +35,7 @@
 -                    int j = armormaterial$layer.dyeable() ? i : -1;
 -                    this.renderModel(p_117119_, p_117120_, p_117123_, p_117124_, j, armormaterial$layer.texture(flag));
 +                net.neoforged.neoforge.client.extensions.common.IClientItemExtensions extensions = net.neoforged.neoforge.client.extensions.common.IClientItemExtensions.of(itemstack);
++                extensions.setupModelAnimations(p_117121_, itemstack, p_117122_, p_117124_, limbSwing, limbSwingAmount, partialTick, ageInTicks, netHeadYaw, headPitch);
```

Comment:
Instead of passing `p_117124_` wouldn't it make more sense to pass in `model` (aka the model that has potentially already been replaced by the modder)?

Reply:
whoops, fixed now